packages/elements/src/account/elements/logto-username.ts

```diff
@@ -0,0 +1,35 @@
+import { html, type TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+
+import usernameIcon from '../icons/username.svg';
+
+import { LogtoProfileItemElement } from './LogtoProfileItemElement.js';
+
+const tagName = 'logto-username';
+
+@customElement(tagName)
+export class LogtoUsername extends LogtoProfileItemElement {
+  static tagName = tagName;
+
+  protected isAccessible() {
+    return this.accountContext?.userProfile.username !== undefined;
```

Comment:
should we display a placeholder here if the username is empty but editable?

Reply:
We have, the `Not set` will be rendered when a username is not set.
<img width="767" alt="image" src="https://github.com/user-attachments/assets/d4e0c9f0-5d5f-4b50-8693-6c169deced5e">